### PR TITLE
Feature/add load more section page

### PR DESCRIPTION
**Pull request by codiebeulaine**

### CODE REVIEW

**Comment by codiebeulaine** [comment]
@saeedmb8 ready for review. @projectmushroom tested locally and works perfectly. Thanks

**Comment by saeedmb8** [comment]
👍

File: molo/core/views.py [hunk 1]
```diff
@@ -275,6 +275,19 @@ def home_index(
...
 @page_template(
+    'patterns/basics/sections/sectionpage-article-list-'
+    'standard_for-paging.html')
+def section_index(
+        request,
+        extra_context=None,
+        template=(
+            'patterns/basics/sections/sectionpage-article-list-'
+            'standard_for-paging.html')):
+    section = SectionPage.objects.get(pk=request.GET.get('section'))
```
Comment (miltontony) [inline molo/core/views.py]: Ideally this should be a `get_object_or_404`

Comment (codiebeulaine) [inline molo/core/views.py]: @miltontony will change in next PR

**Pull request merged**